.chloggen/feat_add-format-identification.yaml

```
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: 'enhancement'
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: 'awslogsencodingextension'
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: 'Add format identification to AWS logs'
```

Add instrumentation scope attributes to AWS logs
```suggestion
note: 'Add instrumentation scope attributes to aws logs'
```